mentat/parsers/block_parser.py

```diff
@@ -149,9 +149,12 @@ def _special_block(
         if ending_line < starting_line:
             raise ModelError("Error: Model output malformed edit.")
 
-        file_lines = self._get_file_lines(
-            code_file_manager, rename_map, deserialized_json.file
+        full_path = (cwd / deserialized_json.file).resolve()
+        rename_file_path = (
+            (cwd / deserialized_json.name).resolve() if deserialized_json.name else None
         )
+
+        file_lines = self._get_file_lines(code_file_manager, rename_map, full_path)
```

Comment:
This is going to lead to a bug when renaming files; _get_file_lines expects a relative path; I'm going to see if I can change the rename map to use absolute paths instead.